### PR TITLE
feat(GitHub): ADDON-44226 Support for GitHub Enterprise

### PR DESCRIPTION
**Pull request by shivanitalreja-splunk**
JIRA ID : https://jira.splunk.com/browse/ADDON-44226

### CODE REVIEW

**Review by samirhadfa-crest — APPROVED** [review]

**Review by rfaircloth-splunk — CHANGES_REQUESTED** [review]
Please change "GITHUB" to "GITHUB_ENT" we prefer "vendor_product" syntax to "vendor"

**Comment by shivanitalreja-splunk** [comment]
> Please change "GITHUB" to "GITHUB_ENT" we prefer "vendor_product" syntax to "vendor"

I have done the suggested changes.

**Comment by shivanitalreja-splunk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Comment by samirhadfa-crest** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Comment by shivanitalreja-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by samirhadfa-crest** [comment]
I have read the CLA Document and I hereby sign the CLA

File: package/etc/conf.d/conflib/syslog/app-github_ent.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+block parser github_ent-parser() {    
+ channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index("gitops")
+                source("github:enterprise:audit")
+                sourcetype('github:enterprise:audit')
+                vendor_product("github_enterprise")
```
Comment (rfaircloth-splunk) [inline package/etc/conf.d/conflib/syslog/app-github_ent.conf:8]: this should match the key in docs GITHUB_ENT (lower case)

**Review by rfaircloth-splunk — CHANGES_REQUESTED** [review]

File: package/etc/conf.d/conflib/syslog/app-github_ent.conf (hunk 1 above)
Comment (shivanitalreja-splunk) [reply package/etc/conf.d/conflib/syslog/app-github_ent.conf]: Yes I have made the change.

**Review by rfaircloth-splunk — APPROVED** [review]

**Pull request merged**